src/components/TableModule/types.ts

```diff
@@ -53,8 +53,8 @@ export interface TableState {
 }
 
 export interface RowSelectionRow {
-  getIsSelected?: () => boolean;
-  getCanSelect?: () => boolean;
-  toggleSelected?: (value?: boolean) => void;
-  getToggleSelectedHandler?: () => (event: unknown) => void;
+  getIsSelected: () => boolean;
```

Comment:
If the prior allows conditional, does this mean that they are all now required?

Reply:
Yes. When using row selection feature, the table configuration MUST be in type `TableConfiguration< RowSelectionRow>`. E.g., https://github.com/lifeomic/chroma-react/blob/84d1c5fe42995c7c24b53c99e98d4674ed6d7043/src/components/TableModule/TableModule.stories.tsx#L457-L471

When not using row selection feature, the table configuration could be in type `TableConfiguration<any>` just what it is today.

Reply:
Love it! 👍🏽